packages/braid-design-system/src/lib/components/Hidden/Hidden.tsx

```diff
@@ -32,18 +33,25 @@ export const Hidden = ({
   data,
   ...restProps
 }: HiddenProps) => {
-  if (process.env.NODE_ENV === 'development' && screen) {
-    // eslint-disable-next-line no-console
-    console.warn(
-      `You used the "screen" prop on Hidden, but this probably doesn't do what you expect. If you're trying to provide content to screen readers without rendering it to the screen, use the <HiddenVisually> component instead. The "screen" prop is likely to be deprecated in a future release.`,
-    );
+  if (process.env.NODE_ENV === 'development') {
+    if (screen) {
```

Comment:
Recommend using a `typeof` check here to capture any usage of the prop.

```suggestion
    if (typeof screen !== 'undefined') {
```